Source/Bogus/Faker[T].cs

```diff
@@ -159,16 +176,108 @@ public virtual Faker<T> UseSeed(int seed)
+      /// </summary>
+      /// <returns></returns>
+      public virtual Faker<T> UseConstructor()
+      {
+         return UseConstructor(false);
       }
 
+
+      /// <summary>
+      /// Will use default <typeparamref name="T"/> public/private/protected parameterless constructor (based on <paramref name="isConstructorHidden"/> ). If parameterless constructor does not exits exception is thrown
+      /// </summary>
+      /// <param name="isConstructorHidden"></param>
+      /// <returns></returns>
+      public virtual Faker<T> UseConstructor(bool isConstructorHidden)
+      {
+
+#if NETSTANDARD1_3
+         throw new NotSupportedException($"{nameof(UseConstructor)}() is not supported in .net standart 1.3. Consider using {nameof(UseConstructor)}().");
+#else
+
+         var constructorModifier = isConstructorHidden ? BindingFlags.NonPublic : BindingFlags.Public;
+         var constructor = typeof(T).GetConstructor(BindingFlags.Instance | constructorModifier, null, Type.EmptyTypes, null);
+         if (constructor == null)
+         {
+            throw new NoParameterlessCtorException<T>(isConstructorHidden, null);
+         }
+
+
+         this.CreateActions[currentRuleSet] = (Faker f) =>
+         {
+            T instance;
+            try
+            {
+               //var constructor = typeof(T).GetConstructor(BindingFlags.Instance | BindingFlags.NonPublic, null, Type.EmptyTypes, null);
+               instance = (T)Activator.CreateInstance(typeof(T), nonPublic: isConstructorHidden);
+            }
+            catch (MissingMethodException ex)
+            {
+               throw new NoParameterlessCtorException<T>(isConstructorHidden, ex);
+            }
+            return instance;
+         };
+
+         return this;
+#endif
+      }
+
+
+
+      /// <summary>
+      /// Creates instance of <typeparamref name="T"/> without calling any constructor. 
+      /// <para/> Use only when atleast one apply: <br/>
+      /// -  <typeparamref name="T"/> is record / struct or <typeparamref name="T"/> does not have any logic in constructor excpect setting properties or public fields <br/>
+      /// -  If you need features that can't be used with <seealso cref="UseConstructor(Func{Faker, T})"/>
+      /// <para/> Consider using <seealso cref="UseConstructor(Func{Faker, T})"/> or <seealso cref="UseHiddenConstructor(Func{Faker, T})"/>
+      /// </summary>      
+      /// <returns></returns>
+      public virtual Faker<T> SkipConstructor()
+      {
+#if NETSTANDARD1_3
+         throw new NotSupportedException($"{nameof(SkipConstructor)}() is not supported in .net standart 1.3. Consider using {nameof(UseConstructor)}().");
+#else
+         //Not sure wich works better
+         //this.CreateActions[Default] = (Faker faker) => (T)FormatterServices.GetUninitializedObject(typeof(T));
+         UseConstructor((Faker faker) => (T)System.Runtime.Serialization.FormatterServices.GetUninitializedObject(typeof(T)));
```

Comment:
Not sure if this.CreateActions[Default] should be called or this.CreateActions[currentRuleSet] instead.